youtube_dl/YoutubeDL.py

```diff
@@ -250,6 +250,10 @@ class YoutubeDL(object):
     call_home:         Boolean, true iff we are allowed to contact the
                        youtube-dl servers for debugging.
     sleep_interval:    Number of seconds to sleep before each download.
+                        Accepts range for random sleep in formats:
+                        - <lower limit>to<upper limit>
+                        - <upper limit>to<lower limit>
+                            where limits are in seconds
```

Comment:
No, AtoB is definitely not how it should look like. It should be pair of options instead.